jwst/assign_wcs/util.py

```diff
@@ -88,7 +90,81 @@ def _reproject(x, y):
     return _reproject
 
 
-def wcs_from_footprints(dmodels, refmodel=None, transform=None, bounding_box=None, domain=None):
+def compute_scale(wcs: WCS, fiducial: Union[tuple, np.ndarray]) -> float:
+    """Compute scaling transform.
+
+    Parameters
+    ----------
+    wcs :
```

Comment:
Ok, sure I can do that. I hesitated since I used annotations in the function definition (I thought that it would be redundant to specify again), but I can change that